setup/db/db/schema-4920to41000.sql

```diff
@@ -232,3 +232,5 @@ WHERE (o.cpu is null AND o.speed IS NULL AND o.ram_size IS NULL) AND
 
 -- CLOUDSTACK-9827: Storage tags stored in multiple places
 DROP VIEW IF EXISTS `cloud`.`storage_tag_view`;
+
+INSERT IGNORE INTO `cloud`.`configuration` VALUES ('Advanced', 'DEFAULT', 'VMSanpshotManager', 'vmsnapshot.expire.interval', '-1', 'VM Snapshot expire interval in hours','-1', NULL, 'Account', 0);
```

Comment:
@abhinandanprateek Since you've used Configkey, no need to use this INSERT query. Also typo in the 'VMSanpshotManager'.